cluster/gce/manifests/cloud-controller-manager.manifest

```diff
@@ -0,0 +1,113 @@
+{
+"apiVersion": "v1",
+"kind": "Pod",
+"metadata": {
+  "name":"cloud-controller-manager",
+  "namespace": "kube-system",
+  "labels": {
+    "tier": "control-plane",
+    "component": "cloud-controller-manager"
+  }
+},
+"spec":{
+"securityContext": {
+  {{runAsUser}}
+  {{runAsGroup}}
+  {{supplementalGroups}}
+  "seccompProfile": {
+      "type": "RuntimeDefault"
+  }
+},
+"priorityClass": "system-node-critical",
```

Comment:
priorityClassName, otherwise this pod gets preempted since priorityClass is not a valid field and gets assigned a priority of 0